benches/nexmark/config.rs

```diff
@@ -37,16 +22,89 @@ pub struct Config {
     pub auction_proportion: usize,
 
     #[clap(
-        long = "bit-proportion",
+        long = "bid-proportion",
         default_value = "46",
         env = "NEXMARK_BID_PROPORTION",
         help = "Specify the proportion of events that will be new bids"
     )]
     pub bid_proportion: usize,
+
+    #[clap(
+        long = "person-proportion",
+        default_value = "1",
+        env = "NEXMARK_PERSON_PROPORTION",
+        help = "Specify the proportion of events that will be new people"
```

Comment:
Help can be given via doc comments, I think that's nicer and it also provides docs with editors that support it
```suggestion
    /// Specify the proportion of events that will be new people
    #[clap(
        long = "person-proportion",
        default_value = "1",
        env = "NEXMARK_PERSON_PROPORTION",
```